spark-derived-feature-view/docker-compose.yaml

```diff
@@ -0,0 +1,73 @@
+version: "2.1"
+services:
+  spark-master:
+    image: feathub-spark:3.3.1
+    container_name: spark-master
+    ports:
+      - "8080:8080"
+      - "7077:7077"
+    depends_on:
+      - datanode
+    environment:
+      - SPARK_LOCAL_IP=spark-master
+      - SPARK_WORKLOAD=master
+
+  spark-worker-a:
+    image: feathub-spark:3.3.1
+    container_name: spark-worker-a
+    ports:
+      - "8081:8080"
+      - "7000:7000"
+    depends_on:
+      - spark-master
+    environment:
+      - SPARK_MASTER=spark://spark-master:7077
+      - SPARK_WORKER_CORES=1
+      - SPARK_WORKER_MEMORY=1G
+      - SPARK_DRIVER_MEMORY=1G
+      - SPARK_EXECUTOR_MEMORY=500M
+      - SPARK_WORKLOAD=worker
+      - SPARK_LOCAL_IP=spark-worker-a
+    volumes:
+       - ./apps:/tmp/spark-apps
+
+  spark-worker-b:
+    image: feathub-spark:3.3.1
+    container_name: spark-worker-b
```

Comment:
Why do we need two workers in this example?

Would it be simpler to start 1 master and 1 worker in the same docker container, similar to what is done in Flink examples?